nym-vpn-core/crates/nym-vpn-api-client/src/jwt.rs

```diff
@@ -120,7 +121,6 @@ impl Jwt {
         }
     }
 
-    #[allow(dead_code)]
     pub fn jwt(&self) -> String {
```

Comment:
Nit: You could return `&self.jwt` instead in here. Cloning can be done by call site if needed.